Umbraco-Cloud/Upgrades/Migrating-from-8-to-10/index.md

```diff
@@ -81,6 +81,23 @@ You need to manually upgrade the view files and custom code implementation. For
 * In Umbraco Forms version 9.0.0+, it is only possible to store Form data in the database. If Umbraco Forms is used on the Umbraco 8 project:
   * Make sure to first migrate the Forms to the database, see the [Umbraco Forms in the Database](../../../Add-ons/UmbracoForms/Developer/Forms-in-the-Database/index-v8) article.
 
+* Add the `global.json` in root repository folder
+:::note
+If `global.json` does not exist in the root of your repository, you can read about the `global.json` [Microsoft docs](https://learn.microsoft.com/en-us/dotnet/core/tools/global-json)
```

Comment:
@mclausen So what do you need to do if the file does not exist in the roof of your Repo?

Do you need to create the file like explained in this [documentation ](https://our.umbraco.com/documentation/Umbraco-Cloud/Troubleshooting/Deployments/#issues-with-net-7-deployments-between-local-and-envs)or?

Reply:
Read the link and create one? :) I am unsure if it's necessary to be too explicit

Reply:
@mclausen Just read it again, and I think I misread it last week 🤦‍♂️

It makes sense to me what you wrote :)